ntbs-service/Startup.cs

```diff
@@ -422,6 +423,7 @@ private static void UseAzureAdAuthentication(IServiceCollection services, AdOpti
                                             context.Options.ClaimsIssuer),
                                         new Claim(ClaimTypes.Role, adOptions.BaseUserGroup, ClaimValueTypes.String),
 
+                                        //new Claim(ClaimTypes.Role, adOptions.ReadOnlyUserGroup, ClaimValueTypes.String),
```

Comment:
We should remove this

Reply:
Ah yeah, this was for basic http auth locally before we had the AD group